apps/cowswap-frontend/src/modules/injectedWidget/updaters/InjectedWidgetUpdater.tsx

```diff
@@ -34,6 +37,12 @@ const cacheMessages = (event: MessageEvent) => {
   messagesCache[method] = event.data
 }
 
+const paramsWithoutPartnerFee = (params: CowSwapWidgetParams) => {
+  delete params.partnerFee
+
+  return params
```

Comment:
This is mutating the original params.
Since it's being returned, would it make sense to make a copy instead?

Something like:

```suggestion
 const { partnerFee: _, ...rest } = params

  return rest
```

Reply:
Oh, nice! Thank you!